cypress/integration/make_new_recipe.js

```diff
@@ -1,7 +1,6 @@
-// todo make test for making new recipe
 describe('Making new test recipe', () => {
   it('Goes to recipe site', () => {
-    // cy.visit('http://localhost:3000/recipe');
+    cy.visit('http://localhost:3000/recipe');
```

Comment:
toto sa vobec pouziva? a ak hej, tak neda sa to cez ten `cy.visit('recipe');` ako mas nizsie?